specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/compute.json

```diff
@@ -8318,6 +8952,253 @@
+            "$ref": "#/definitions/CapacityReservationInstanceViewWithName"
+          },
+          "readOnly": true,
+          "description": "List of instance view of the capacity reservations under the capacity reservation group."
+        }
+      }
+    },
+    "CapacityReservationGroupProperties": {
+      "type": "object",
+      "properties": {
+        "capacityReservations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SubResourceReadOnly"
+          },
+          "readOnly": true,
+          "description": "A list of references to all capacity reservations in the capacity reservation group."
+        },
+        "virtualMachinesAssociated": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SubResourceReadOnly"
+          },
+          "readOnly": true,
+          "description": "A list of references to all virtual machines associated to the capacity reservation group."
+        },
+        "instanceView": {
+          "$ref": "#/definitions/CapacityReservationGroupInstanceView",
+          "readOnly": true,
+          "description": "The capacity reservation group instance view, which has the list of instance view of the capacity reservations under the capacity reservation group."
+        }
+      },
+      "description": "capacity reservation Group Properties."
+    },
+    "CapacityReservationGroup": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CapacityReservationGroupProperties"
+        },
+        "zones": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Availability Zones to use for this capacity reservation group. The zones can be assigned only during creation. If not provided, the group supports only regional resources in the region. If provided, enforces each capacity reservation in the group to be in one of the zones."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "description": "Specifies information about the capacity reservation group that the capacity reservations should be assigned to. <br><br> Currently, a capacity reservation can only be added to a capacity reservation group at creation time. An existing capacity reservation cannot be added to another capacity reservation group."
+    },
+    "CapacityReservationGroupUpdate": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CapacityReservationGroupProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/UpdateResource"
+        }
+      ],
+      "description": "Specifies information about the capacity reservation group that the capacity reservation should be assigned to. Only tags can be updated."
```

Comment:
Update description to:

Specifies information about the capacity reservation group. Only tags can be updated.

Reply:
updated